bite-size-notes/week-8-d-Probability and Recursive Bayesian Filtermd.md

```diff
@@ -0,0 +1,210 @@
+# Probability and Recursive Bayesian Filter
```

Comment:
Overall, I think the notes provide the reader with a good basis to understand Bayes theorem and the Recursive Bayesian Filter. I think there could be some improvements to the descriptions of each topic. Also adding an example or code demo of how these concepts can be used would be beneficial.

Reply:
Thank you for your time, I will pay more attention to these mistakes later.